mnefun/_report.py

```diff
@@ -218,11 +218,16 @@ def gen_html_report(p, subjects, structurals, run_indices=None,
                             info=sss_info, subjects_dir=subjects_dir, bem=bem,
                             dig=True, coord_frame=coord_frame, show_axes=True,
                             fig=fig, trans=trans, src=src)
-                        try:
-                            mne.viz.plot_alignment(surfaces='head-dense',
-                                                   **kwargs)
-                        except Exception:
-                            mne.viz.plot_alignment(surfaces='head', **kwargs)
+                        try_surfs = ['head-dense', 'head', 'inner_skull']
+                        for surf in try_surfs:
+                            try:
+                                mne.viz.plot_alignment(surfaces=surf, **kwargs)
+                            except Exception:
+                                pass
+                        else:
+                            raise RuntimeError('Could not plot any surface '
+                                               'for alignment:\n%s'
+                                               % (try_surfs,))
```

Comment:
Not sure what we want here WRT to `fsaverage` as the surrogate anatomy.  Am I missing something?

> Traceback (most recent call last):
>   File "/home/ktavabi/Projects/Andrea-Anders/andrea_anders/mnefun_process.py", line 114, in <module>
>     print_status=True
>   File "/home/ktavabi/Projects/mnefun/mnefun/_mnefun.py", line 615, in do_processing
>     outs[ii] = func(p, subjects, structurals, run_indices)
>   File "/home/ktavabi/Projects/mnefun/mnefun/_report.py", line 230, in gen_html_report
>     % (try_surfs,))
> RuntimeError: Could not plot any surface for alignment:
> [u'head-dense', u'head', u'inner_skull']

Reply:
Have you updated? It should work with the latest commit.